### ISSUE TITLE
Builder Cannot Reuse Banners he Modified

### ISSUE THREAD

**Issue by desagas** [ldtteam/minecolonies#6973]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.5.36.1.2
- MineColonies Version: 110.Alpha
- Structurize Version: 166-ALPHA
- Related Mods and their Versions: ATM6-1.6.1

## Expected behavior

When building with banners that came from a deconstructed build, back onto the exact same replaced build, the builder should be able to reuse the banners that were on the building before hand, instead of requesting new ones be crafter and wasting wool.

## Actual behavior

Builder requests new banners everytime a structure is moved, even if the exact same banners exist in his or his huts inventory.

![2021-04-22_00 08 18](https://user-images.githubusercontent.com/8404632/115664361-f4f4aa00-a2fe-11eb-9786-d4e19250e375.png)
![2021-04-22_00 08 38](https://user-images.githubusercontent.com/8404632/115664375-f9b95e00-a2fe-11eb-9262-1034650cf29c.png)
![2021-04-22_00 08 52](https://user-images.githubusercontent.com/8404632/115664380-f9b95e00-a2fe-11eb-830e-63ad367be5ec.png)

## Steps to reproduce the issue

Construct and deconstruct Level 5 Caledonia Mystical Site. Builder will request new banners instead of using old ones.

1. 
2. 
3. 

### Logs

- latest.log: 
- crashlog: 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Raycoms** [comment]
Yeah this is heavily related to the issue you mentioned. As of now you have to do this manually in a cauldron. It's a limitation of the current system and as such not a bug. We have plans in the future to handle this better. So I am moving it to the feature repo.
